cloudify/context.py

```diff
@@ -557,7 +557,7 @@ def __init__(self, ctx=None):
         self._logger = None
         self._provider_context = None
         self._bootstrap_context = None
-        self._host_ip = None
+        self._host_ip = self._context.get('rest_host')
```

Comment:
@mabuaisha i see you renamed to `rest_host` but still this `self._host_ip` is used anyway
but this existed beforehand, no?
probably better let's add a new attribute instead of reusing that one, i don't know if it is used by anything else?
i'd say `self._rest_host` for safety, and if `self._host_ip` is not used, then delete it

Reply:
I totoally agree with you, will use another one instead of this because it could be used by another one

Reply:
I cannot see any reference to `_host_id` used so I think we can remove it